Cargo.toml

```diff
@@ -53,12 +53,12 @@ wgpu-profiler = { workspace = true, optional = true }
 # Note: we pick up a kurbo dependency transitively through peniko.
 # However, we need at least version 0.10.4 for proper stroking.
 # Next time we bump peniko, this can be removed.
-kurbo = "0.10.4"
+kurbo = "0.11.0"
```

Comment:
The one thing I notice is this "Next time we bump peniko, this can be removed." comment.

Reply:
Yeah I was looking into that.

Reply:
Works fine without it.